test/multiverse/suites/active_record/Envfile

```diff
@@ -18,14 +18,16 @@ RAILS_VERSIONS = [
   ['3.2.0', 2.2, 2.2]
 ]
 
-def jbdc_version(activerecord_version)
+def jdbc_version(activerecord_version)
   return unless RUBY_PLATFORM.eql?('java') || activerecord_version.nil?
-  activerecord_version.slice('1')
+  # remove the '.' between the major and minor versions
+  activerecord_version_jdbc = activerecord_version.sub('.', '')
+  activerecord_version_jdbc
```

Comment:
No need for a variable here, just a line with `activerecord_version.sub('.', '')` by itself will work